src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLFeaturesBuilder.java

```diff
@@ -230,4 +290,38 @@ private boolean getForcedDecimal(ResourceInfo meta) {
         }
         return false;
     }
+
+    private class FeatureSourceSimplifier {
```

Comment:
Looks almost like a leftover... all that is used here is the query property, everything else is collected but then not used.

Reply:
Removed and replaced with just query.